src/components/homepage/SmartSignalsSection/SmartSignalsSection.tsx

```diff
@@ -1,16 +1,52 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Container from '../../common/Container'
 import Button from '../../common/Button'
 import { PATH, URL } from '../../../constants/content'
 
 import styles from './SmartSignalsSection.module.scss'
 
 import { Link } from 'gatsby'
+import classNames from 'classnames'
 
 import { ReactComponent as SmartSignalsSVG } from './SmartSignalsSVG.svg'
 import { ReactComponent as ArrowSVG } from './ArrowSVG.svg'
 
+import { ReactComponent as BotDetectionSVG } from './img/BotDetectionSVG.svg'
+import { ReactComponent as UserBehaviorSVG } from './img/UserBehaviorSVG.svg'
+import { ReactComponent as DeviceSVG } from './img/DeviceSVG.svg'
+import { ReactComponent as NetworkSVG } from './img/NetworkSVG.svg'
+import { ReactComponent as MobileSVG } from './img/MobileSVG.svg'
+
+import { motion, AnimatePresence } from 'framer-motion'
+
+enum SmartSignals {
+  BotDetection = 'bot-detection',
+  UserBehaviorSignals = 'user-behavior-signals',
+  DeviceSignals = 'device-signals',
+  NetworkSignals = 'network-signals',
+  MobileSignals = 'mobile-signal',
+}
+
 export default function SmartSignalsSection() {
+  const [selectedCard, setSelectedCard] = useState<SmartSignals>(SmartSignals.BotDetection)
```

Comment:
Nit: ts should be able to infer the type here.
```suggestion
  const [selectedCard, setSelectedCard] = useState(SmartSignals.BotDetection)
```